### PR TITLE
Added SnapShooter Backup Service to Others

### PR DESCRIPTION
**Pull request by mrsimonbennett**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1088,3 +1088,4 @@ Table of Contents
   * [github.com — FOSS for Dev](https://github.com/tvvocold/FOSS-for-Dev) — A hub of free and Open Source software for developers 
   * [smsreceivefree.com](https://smsreceivefree.com/) — Provides free temporary and disposable phone numbers.
   * [simplebackups.io](https://simplebackups.io/) — Backup automation service for servers and databases (MySQL, PostgreSQL, MongoDB) stored directly into cloud storage providers (AWS, DigitalOcean, Backblaze...). Provides free plan for 1 backup.
+  * [SnapShooter](https://snapshooter.io/) — Backup solution for DigitalOcean, AWS, LightSail, Hetzner and Exoscale, with support for direct database, file system and application backups to s3 based storage.
```
Comment (ripienaar) [inline README.md:1091]: looks good, would you mind mentioning something like 'free for 1 backup' or so?

Comment (mrsimonbennett) [reply README.md]: Resolved @ripienaar :) Thanks

**Comment by ripienaar** [comment]
thanks

**Comment by mrsimonbennett** [comment]
Thanks @ripienaar 🙏

**Pull request merged**